### ISSUE TITLE
chaim1221/Machete cannot add role to user

### ISSUE THREAD

**Issue by chaim1221** [SavageLearning/Machete#417]
```
An unhandled exception occurred while processing the request.
Exception: AccountController, UserRoles method, `result` failed: System.Collections.Generic.List`1[Microsoft.AspNetCore.Identity.IdentityError]
Machete.Web.Controllers.AccountController.UserRoles(SelectUserRolesViewModel model) in AccountController.cs, line 445
```
diagnose, fix

**Comment by chaim1221** [comment]
tackling this along with #414

**Comment by chaim1221** [comment]
just appending to this issue...create user works, but if you put in an invalid password it fails silently without an exception

**Comment by chaim1221** [comment]
this is resolved by chaim1221#2
